sensor.go

```diff
@@ -0,0 +1,98 @@
+/*
+* sensor.go
+* GoHue library for Philips Hue
+* Copyright (C) 2016 Collin Guarino (Collinux) collin.guarino@gmail.com
```

Comment:
This is a new file, so you probably can have your own copyright information here. At least the year should be current.

Reply:
This is just copied from #13 so, probably @bboehmke's details should go there :)